src/SFA.DAS.Reservations.Application/Reservations/Commands/CreateReservationApiRequest.cs

```diff
@@ -0,0 +1,10 @@
+﻿using System;
+
+namespace SFA.DAS.Reservations.Application.Reservations.Commands
+{
+    public class CreateReservationApiRequest
```

Comment:
Can this be moved - would be better in the domain

Also if we could change it to be like:
```
public class CreateReservationApiRequest
    {
        public CreateReservationApiRequest(Func<string,long> setAccountId, string employerAccountId, DateTime startDate)
        {
            AccountId = setAccountId(employerAccountId);
            StartDate = startDate;
        }
        public long AccountId { get; }
        public DateTime StartDate { get;}
    }
```
Where the setAccountId is the _hashingService.Decode